dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/metadata/MetadataPresenter.java

```diff
@@ -18,33 +20,44 @@
 import java.util.Optional;
 import java.util.ResourceBundle;
 
+@Presenter
 public class MetadataPresenter implements Initializable {
 
     private final static Logger LOG = LoggerFactory.getLogger(MetadataPresenter.class);
     private final SimpleDateFormat formatter = new SimpleDateFormat(StringConstants.DATE_FORMAT);
     private final Calendar calendar = Calendar.getInstance();
+
     @FXML
     private Label objectName;
+
     @FXML
     private Label objectSize;
 
     @FXML
     private Label lastModified;
+
     @FXML
     private Tooltip nameTooltip;
+
     @FXML
     private TableView<MetadataEntry> metadataTable;
+
     @FXML
     private TableColumn<MetadataEntry, String> metadataTableColValue;
 
     @FXML
     private TableColumn metadataTableColName;
 
-    @Inject
-    private Ds3Metadata ds3Metadata;
+    @ModelContext
```

Comment:
Is this injected from the view?  If so it should not be passed in the constructor, and should be made non-final.